website/docs/basics/blocks-styles.md

```diff
@@ -510,3 +561,229 @@ Attribute value replacement variable is used to return the attribute value where
 	}
 }
 ```
+
+## Responsive variables
+
+What this option does is reduce multiplying the code. Let's say you have 3 breakpoints and for each breakpoint, since it can be set separately, you have to have 3 attributes. To avoid setting the same variables for each attribute, you can map all of the attribute names into one under an overall key(in the example down below `wrapperHide`) in `responsiveAttributes` property in manifest. Mapping should be done by adding `breakpoint` as a key (let's say `large`) and an attribute name (in this case `wrapperHideLarge`) as a value. The overall key can then be used in `variables` as a template for the "real" attributes (this can also be used in `variablesEditor`).
```

Comment:
multiplying? Or did you mean duplicating?

```suggestion
What this option does is reduce multiplying the code. Let's say you have 3 breakpoints and for each breakpoint, since it can be set separately, you need to have 3 attributes. To avoid setting the same variables for each attribute, you can map all of the attribute names into one under an overall key (in the example down below it's `wrapperHide`) in `responsiveAttributes` property in the manifest. Mapping should be done by adding `breakpoint` as a key (let's say `large`) and an attribute name (in this case `wrapperHideLarge`) as a value. The overall key can then be used in `variables` as a template for the "real" attributes (this can also be used in `variablesEditor`).
```

Reply:
What I was thinking is to reduce the number of copy-pasting the same code multiple times. Maybe duplicating could be the right word. I was thinking more in a sense of using it multiple times and not 2 times. But I agree that it might not be the right usage of the word. 😄 

Would you prefer to use duplicating?

Reply:
Yeah, that's like the definition of duplication (avoiding copy/paste) 😂

Reply:
Ok, sorry. I'm being ignorant.